src/views/AnnualReport.vue

```diff
@@ -914,6 +937,7 @@ export default {
 
     resetErrors () {
       this.saveErrorDialog = false
+      this.bcolErrorDialog = false
```

Comment:
Obsolete code.

Also in other 2 filings.

Reply:
done

Reply:
Fixed.